fastmot/utils/visualization.py

```diff
@@ -1,14 +1,22 @@
 import colorsys
 import numpy as np
 import cv2
-
+from .rect import get_center
 
 GOLDEN_RATIO = 0.618033988749895
 
+def draw_trajectories(frame, track):
+    tlbrs = np.reshape(list(track.bboxes), (len(track.bboxes), 4))
+    centers = tuple(map(lambda box: get_center(box), tlbrs[::4]))
+    color = get_color(track.trk_id)
+    pts = np.array(centers, dtype=np.int32)
+    cv2.polylines(frame, [pts], False, color, thickness=1)
 
-def draw_tracks(frame, tracks, show_flow=False, show_cov=False):
+def draw_tracks(frame, tracks, show_flow=False, show_cov=False, show_traj=False):
     for track in tracks:
         draw_bbox(frame, track.tlbr, get_color(track.trk_id), 2, str(track.trk_id))
+        if show_traj:
+            draw_trajectories(frame,track)
         if show_flow:
             draw_feature_match(frame, track.prev_keypoints, track.keypoints, (0, 255, 255))
         if show_cov:
```

Comment:
A few formatting issues before merging: 
- Please follow PEP8 style: two blank lines between each function and global variable. Add space between parameters when calling `draw_trajectories()`. 
- Move draw_trajectories() definition after draw_detections() since it is a helper function.
- Finally, it would be better if draw_trajectories() only takes in a `trk_bboxes` argument instead of the entire track object like the other helper functions.

Reply:
Thank you for all of these info and sorry for the inconvenience! Also, thank you for this great project!

Reply:
Thanks for the pull request.